### DISCUSSION TITLE
Feature request - histogram or column chart with data-based variable width

### DISCUSSION THREAD

**Discussion by SteveRBeeks** [grafana/grafana#44011] [Dashboards, Panels & Visualizations]
<img width="1615" alt="Screenshot 2022-01-13 at 17 12 01" src="https://user-images.githubusercontent.com/86779366/149376740-09dc2e4d-c804-479b-aa23-202c2bb0241e.png">

Producing data into HDR - [http://hdrhistogram.org](http://hdrhistogram.org) - histograms we have exponentially increasing bucket sizes. We currently visualise this with variable-width column charts in HighCharts.

It would be useful to be able to visualise data like this in a similar way.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
